scripts/main.js

```diff
@@ -72,3 +72,85 @@ const departments = {
 }
 
 console.log(departments);
+
+// ------- OPDRACHT 1 -------//
+
+console.log("De afdeling Sales heeft " + departments.sales.numberOfEmployees + " medewerkers");
+console.log("Marketing is een leuke afdeling om te werken. " + departments.marketing.description);
+console.log("De afdeling Customer Service heeft " + departments["customer-service"].numberOfEmployees + " medewerkers");
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. " + departments.sales.jobs[0].description);
+
+// ------- OPDRACHT 2 -------//
+const userInput = prompt("Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]");
+
+console.log(userInput);
+
+if (userInput === "marketing" || userInput === "sales" || userInput === "customer-service") {
+    console.log("Je koos " + userInput + ": " + departments[userInput].description);
+} else {
+    // console.error("Ongeldige keuze")
+    document.getElementById('error-message').textContent = "Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.";
+}
+
+// ------- OPDRACHT 4 -------//
+
+console.log(userInput + " is een leuke afdeling om te werken. Er werken op dit moment " + departments[userInput].numberOfEmployees + " medewerkers.");
+
+
+// ------- OPDRACHT 3 -------//
+
+let numberOfJobs = "";
+
+for (let i = 0; i < departments[userInput].jobs.length; i++) {
+    numberOfJobs += i + ": " + departments[userInput].jobs[i].title + "\n";
+}
+console.log(numberOfJobs);
+
+const jobDescription = prompt("Je koos " + userInput + ". \nOver welke functie wil je meer weten? Voer een getal tussen 0 en " + (departments[userInput].jobs.length - 1) + " in. \n" + numberOfJobs);
+
```

Comment:
Erg slim om hier een for-loop te gebruiken, hier heb ik dus flink op zitten worstelen en ik wist het bestaan van for-loops nog niet :) hij werkt perfect!